docs/en/setup/backend/aws-firehose-receiver.md

```diff
@@ -18,3 +18,10 @@ CloudWatch metrics with S3 -->  CloudWatch Metric Stream (OpenTelemetry formart)
 
 1. Only OpenTelemetry format is supported (refer to [Metric streams output formats](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-metric-streams-formats.html))
 2. Only HTTPS could be accepted, you could directly enable TLS and set the receiver to listen 443, or put the receiver behind a gateway with HTTPS (refer to [Amazon Kinesis Data Firehose Delivery Stream HTTP Endpoint Delivery Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html))
+
+The OTEL proto in this module copy from [opentelemetry-proto v0.7.0](https://github.com/open-telemetry/opentelemetry-proto/tree/v0.7.0) as AWS firehose only support OTEL 0.7, and rewrite follows :
+
+1. Only remain metrics part
+2. Change `java_package` to avoid class conflict with existing OTEL proto (oap-server/server-receiver-plugin/receiver-proto)
+
+To reduce change of `otel-receiver-plugin` and reuse MAL, `org.apache.skywalking.oap.server.receiver.aws.firehose.OtelMetricsConvertor` converts OTEL v0.7.0 ExportMetricsServiceRequest to existing OTEL proto (oap-server/server-receiver-plugin/receiver-proto).
```

Comment:
```suggestion
```

We don't need to mention this(why we wrote codes in this way) in the doc.
If you prefer to be clear, left this in PR description in case of ppl want to know the context.